src/rust/src/error.rs

```diff
@@ -10,6 +10,7 @@ pub enum Error {
     IllegalHeaderSize(usize),
     InvalidFlatbuffer(InvalidFlatbuffer),
     IO(std::io::Error),
+    UnsupportedGeometryType(String),
```

Comment:
Note that adding a new error variant is a breaking change unless the enum is marked as non-exhaustive.